sound/soc/sof/ipc4.c

```diff
@@ -564,6 +565,15 @@ u32 sof_ipc4_find_debug_slot_offset_by_type(struct snd_sof_dev *sdev, u32 slot_t
 }
 EXPORT_SYMBOL(sof_ipc4_find_debug_slot_offset_by_type);
 
+static inline void sof_ipc4_create_exception_debugfs_node(struct snd_sof_dev *sdev)
+{
+	/* remove the first u32 separator code for slot size */
+	snd_sof_debugfs_add_region_item(sdev, SOF_FW_BLK_TYPE_SRAM,
+					sdev->debug_box.offset,
```

Comment:
It will be updated to correct value when panic happens.  The initialized value is not useful and just a valid value to make snd_sof_debugfs_add_region_item happy. Do you have any suggestion ?